pom.xml

```diff
@@ -325,6 +326,16 @@
 	                    </execution>
 	                </executions>
 	            </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-surefire-plugin</artifactId>
+                    <version>${maven.surefire.plugin.version}</version>
+                    <configuration>
+                        <systemPropertyVariables>
+                            <projectVersion>${project.version}</projectVersion>
```

Comment:
wouldn't it make more sense to define here <mule.maven.plugin.version>
And then in MojoTest
String projectVersion = System.getProperty("mule.maven.plugin.version");

To define projectVersion as project.version will lead to someone mistaking the reason of the property being there in the first place